sentry/src/main/java/io/sentry/ExternalOptions.java

```diff
@@ -32,6 +32,7 @@
   private final @NotNull List<String> inAppExcludes = new CopyOnWriteArrayList<>();
   private final @NotNull List<String> inAppIncludes = new CopyOnWriteArrayList<>();
   private final @NotNull List<String> tracingOrigins = new CopyOnWriteArrayList<>();
+  private final @NotNull List<String> mdcTags = new CopyOnWriteArrayList<>();
```

Comment:
Can anyone think of a name that's a bit more generic than MDC Tags?
In case any other SDK wants to add support for a similar feature it might be nice if they can use the same name.
Something like `logContextTags` or `promoteLogContextEntriesToTags`?

We should then probably mention somewhere that this takes MDC entries and puts them into sentry tags so people can actually find this feature when searching for MDC.

Reply:
Perhaps `contextTags`? I am open to change it to anything more suitable.

Reply:
`contextTags` sounds good to me. Can you please rename it? Sorry for the inconvenience but I'd rather change it now than deprecate and rename later as we've already been asked to pick a name that can be used for more than just the Java SDK.

Reply:
Done.